Dockerfile

```diff
@@ -52,6 +52,6 @@ VOLUME /pct/plugin-src
 VOLUME /pct/jenkins.war
 VOLUME /pct/out
 VOLUME /pct/tmp
-VOLUME /pct/m2-settings.xml
+VOLUME /pct/m2-settings
```

Comment:
It would break compatibility of some my flows based on this image. Why don't we check the file type in `run-pct`? You can check whether it is a folder or not before invoking PCT